kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -355,6 +355,15 @@ object KyuubiConf {
     .transform(_.toLowerCase(Locale.ROOT))
     .createWithDefault(SaslQOP.AUTH.toString)
 
+  val SESSION_SUBMIT_LOG_RETAIN_MILLIS: ConfigEntry[Long] =
+    buildConf("session.submit.log.retain.millis")
```

Comment:
can we just build the conf with `session.engine.log.timeout`

as we have 

session.engine.idle.timeout
session.engine.login.timeout
session.engine.initialize.timeout

etc

It's much easier to group them in the documentation and in the user's mind.

Reply:
yeah see it.